griptape/tools/aws_s3_client/tool.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import annotations
+import boto3
+from schema import Schema, Literal
+from attr import define, field, Factory
+from griptape.artifacts import BaseArtifact, TextArtifact, ErrorArtifact
+from griptape.core.decorators import activity
+from griptape.tools import BaseAwsClient
+
+@define
+class AwsS3Client(BaseAwsClient):
+    session: boto3.session = field(kw_only=True)
+    s3_client: boto3.client = field(
+        default=Factory(lambda self: self.session.client("s3"), takes_self=True)
+    )
+
+    @activity(config={
+        "description": "Retruns the access control list(ACL) of an bucket.",
+        "schema": Schema({
+            Literal(
+                "Bucket",
+                description="The bucket name that contains the object for which to get the ACL information."
```

Comment:
Make this sound simpler for the LLM. Something like "Name of the AWS S3 bucket for which to get an ACL."